src/app/views/account/profile/ProfileController.js

```diff
@@ -84,7 +84,7 @@
           label: 'Personal Website',
           minLength: 5,
           value: 'vm.userEdit.url',
-          helpText: 'Your personal website/blog.'
+          helpText: 'Your personal website/blog. Please only include the domain and path, omitting the \'http://\' protocol part.'
```

Comment:
This forces HTTPS: https://github.com/griffithlab/civic-client/blob/57e591be2c3f6ca1585561f7a1409c87a1fb4aaa/src/components/directives/userCard.tpl.html#L115

So should this help text match that?

```suggestion
          helpText: 'Your personal website/blog. Please only include the domain and path, omitting the \'https://\' protocol part.'
```

Reply:
In general it's better to use https wherever possible so that's why the link out uses it. Websites will switch from https to http if they don't have certs configured for https*, so it seemed to me to be the best choice: security is preserved with https which transparently redirects to http.

* and if the sysadmin has a bare minimum of experience - I'm assuming that hardly anyone maintains their own personal webservers anymore and that the vast majority of the links supplied here will be to personal site hosting services that have competent sysadmins